foundation-extensions/mobile-core/mobile-core-api-reference.md

```diff
@@ -173,21 +171,22 @@ MobileCore.trackAction("loginClicked", additionalContextData);
 **Syntax**
 
 ```swift
-+ (void) trackAction: (nullable NSString*) action data: (nullable NSDictionary*) contextData;
+@objc(trackAction:data:)
+static func track(action: String?, data: [String: Any]?)
 ```
 
 * _action_ contains the name of the action to track.
-* _contextData_ contains the context data to attach on this hit.
+* _contextData_ contains the context data to attach on the hit.
 
 **Example**
 
 ```swift
-ACPCore.trackAction("action name", data: ["key": "value"])
+ACPCore.track(action: "action name", data: ["key": "value"])
```

Comment:
should be `MobileCore.track...` instead of `ACPCore.track...`

Reply:
This section is for calling ACP SDK APIs in Swift code, we should revert changes to the previous version. For using AEP SDK, I have a separate PR for it. https://github.com/Adobe-Marketing-Cloud/aep-sdks-documentation/pull/536